pkgs/shells/fish/plugins/sdkman-for-fish.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "sdkman";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "reitzig";
+    repo = "sdkman-for-fish";
+    rev = "v${version}";
+    sha256 = "sha256-bfWQ2al0Xy8bnJt5euziHz/+qhyri4qWy47VDoPwQcg=";
```

```suggestion
    hash = "sha256-bfWQ2al0Xy8bnJt5euziHz/+qhyri4qWy47VDoPwQcg=";
```
SRI hash is preferable.